main/pcre2/APKBUILD

```diff
@@ -1,18 +1,18 @@
 # Contributor: Jakub Jirutka <jakub@jirutka.cz>
 # Maintainer: Jakub Jirutka <jakub@jirutka.cz>
 pkgname=pcre2
-pkgver=10.30
+pkgver=10.31
 pkgrel=0
 pkgdesc="Perl-compatible regular expression library"
 url="http://pcre.sourceforge.net/"
 arch="all"
 license="BSD"
```

Comment:
What kind of BSD license are they using? Alpine uses [SPDIX license names](https://spdx.org/licenses/) these days. It would imho be better to use the proper BSD license name for this license and not install the LICENSE to `/usr/share/licenses`.

Reply:
Thank you for the heads-up. Also, is the above method preferred in packages that don't include a license file in docs? It would be useful to know from now on.